torch/onnx/_internal/diagnostics/infra/_infra.py

```diff
@@ -272,5 +273,8 @@ class DiagnosticOptions:
     Options for diagnostic context.
     """
 
+    verbosity_level: int = dataclasses.field(default=logging.INFO)
```

Comment:
Do we plan to have another log level outside the context of diagnotic or it is one for all exporter?

If we need only one:
* could we call it `"onnx"` instead of `"onnx_diagnostic"` Given this string is passed to `TORCH_LOGS`, the `_diagnostics` seems redundant
* should it live outside of the context of `DiagnosticOptions`?

Reply:
This is for diagnostic verbosity level, hence under DiagnosticOptions. The public api on ExportOptions can be bikeshedded.

Please checkout [NOTE: `dynamo_export` diagnostics logging] and PT2 logging for log vs artifact log.

"onnx" is the root logger for anything under `torch.onnx`, "onnx_diagnostics" is the artifact logger under "onnx" logger used by diagnostics.

I'm keeping them separate such that

- It is clear when using `TORCH_LOGS="onnx_diagnostics"` it is the diagnostic verbosity level being adjusted.
- "onnx" root logger can be used for tools and testing, places where export is not called and hence no diagnostic around.